build_tools/github_actions/configure_ci.py

```diff
@@ -352,6 +352,10 @@ def main(base_args, linux_families, windows_families):
     #     * workflow_dispatch or workflow_call with inputs controlling enabled jobs?
     enable_build_jobs = should_ci_run_given_modified_paths(modified_paths)
 
+    # In the case of a scheduled run, we always want to build
+    if is_schedule:
+        enable_build_jobs = True
```

Comment:
How about

```python
if is_schedule:
  enable_build_jobs = True
else:
  modified_paths = get_modified_paths(base_ref)
  print("modified_paths (max 200):", modified_paths[:200])
  enable_build_jobs = should_ci_run_given_modified_paths(modified_paths)
```

Easier to see the code paths that way and avoids performing computations that get discarded.